docs/en/latest/building-apisix.md

```diff
@@ -75,7 +75,7 @@ If you get an error message like `Could not find header file for LDAP/PCRE/opens
 
 `luarocks` supports custom compile-time dependencies (See: [Config file format](https://github.com/luarocks/luarocks/wiki/Config-file-format)). You can use a third-party tool to install the missing packages and add its installation directory to the `luarocks`' variables table. This method works on macOS, Ubuntu, CentOS, and other similar operating systems.
 
-The solution below is for macOS but it works similarly for other operating systems:
+The solution below is for macOS but it works similarly for other operating systems, and macOS is not suggested:
```

Comment:
Have trouble understanding this. It says `the solution below is for macOS` but `macOS is not suggested (recommended)`?

Reply:
@kayx23 I followed his advice.
![image](https://github.com/apache/apisix/assets/112474703/2cf8c4b8-f953-4219-bba1-4ea3cb14dfdc)


I don’t know how to modify it next, so I closed it first.